packages/neuron-ui/src/components/Receive/index.tsx

```diff
@@ -73,7 +74,16 @@ const Receive = ({ onClose, address }: { onClose?: () => void; address?: string
   }
 
   const { isInShortFormat, setIsInShortFormat, address: showAddress } = useSwitchAddress(accountAddress)
-  const { ref, onCopyQrCode, onDownloadQrCode, showCopySuccess } = useCopyAndDownloadQrCode()
+  const { ref, onDownloadQrCode, showCopySuccess } = useCopyAndDownloadQrCode()
+
+  const onCopyAddress = useCallback(() => {
+    window.navigator.clipboard.writeText(showAddress)
+    setIsCopySuccess(true)
+
+    setTimeout(() => {
```

Comment:
It's better to cancel the timer when `onTimeout` is called more than one time. Otherwise, the callback of the first `timeout` will set `isCopySuccess` earlier than expected.